viz_scripts/generic_metrics.ipynb

```diff
@@ -25,12 +25,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "year = 2020\n",
-    "month = 11\n",
+    "year = None\n",
+    "month = None\n",
```

Comment:
Sure. I will revert it back to None.

Reply:
You cannot revert it back to `None` because it wasn't `None` to begin with.
Please **check the history** and understand the reason for the reversion.

Reply:
Apology for the typo, I'd revert it back to the Year and Month previously specified.

Reply:
Can you also, for the record, fill in the reason why this doesn't work (that you found from the blame)?

Reply:
In reference to the comment from the commit, [Explanation of why not to fill up the year and month with None](https://github.com/e-mission/em-public-dashboard/commit/63b8a94083e9edb1378bf760edf9c47768b4785a) - "Apparently, `nbparameterise` doesn't work properly if they are None"